File: c/heart.c
#include <stdio.h> 

int main () { 

  for (int row = 0; row < 6; row++) { 
    for (int col = 0; col < 7; col++) { 
      if ((row==0 && col%3!= 0) || (row==1 && col%3==0) || (row-col==2) || (row+col== 8)) { 
        printf(" * ");
      } else { 
        printf("  ");
      }
    }
    printf("\n"); 
  }

  return 0;
}

File: java/Heart.java
public class Heart {

  public static void main(String[] args) { 
    for(int row = 0; row < 10 ; row++) { 
      for(int col = 10; col > 0; col--) {
        if (row - col > 0) { 
          System.out.print(" *");
        } else { 
          System.out.print(" ");
        }
      }
      System.out.println();
    }
  }
}
